src/module/fvttModuleAPIs.js

```diff
@@ -1,24 +1,57 @@
 export const registerFVTTModuleAPIs = () => {
   // see docs for more info https://github.com/fantasycalendar/FoundryVTT-ItemPiles/blob/master/docs/api.md
   Hooks.once("item-piles-ready", async function () {
-    if (ItemPiles.API.ACTOR_CLASS_TYPE !== "character")
-      ItemPiles.API.setActorClassType("character");
-    if (ItemPiles.API.ITEM_QUANTITY_ATTRIBUTE !== "data.quantity.value")
-      ItemPiles.API.setItemQuantityAttribute("data.quantity.value");
-    if (
-      ItemPiles.API.ITEM_FILTERS !==
-      [
-        {
-          path: "type",
-          filters: "spell,ability",
+
+    if(game.itempiles.API.ACTOR_CLASS_TYPE !== "character"){
+      await game.itempiles.API.setActorClassType("character");
+    }
+
+    if (game.itempiles.API.ITEM_QUANTITY_ATTRIBUTE !== "system.quantity.value") {
+      await game.itempiles.API.setItemQuantityAttribute("system.quantity.value");
+    }
+
+    if (game.itempiles.API.ITEM_PRICE_ATTRIBUTE !== "system.cost") {
+      await game.itempiles.API.setItemPriceAttribute("system.cost");
+    }
+
+    const filters = [{
+      path: "type",
+      filters: "spell,ability",
+    }];
+    if (JSON.stringify(game.itempiles.API.ITEM_FILTERS) !== JSON.stringify(filters)) {
+      await game.itempiles.API.setItemFilters(filters);
+    }
+
+    const similarities = ["type", "type"];
+    if (JSON.stringify(game.itempiles.API.ITEM_SIMILARITIES) !== JSON.stringify(similarities)) {
+      await game.itempiles.API.setItemSimilarities(similarities);
+    }
+
+    const currencies = [{
```

Comment:
Should we add silver/copper/electrum/platinum here, as well?

Reply:
Here's the context for this comment @Haxxer 

The game system assumes a default currency in exactly one place: in character creation GP is used, because rolling for GP is a step in OSE SRD's character creation. The game system code `ose` makes no other reference to denominations.

The compendium `ose-content` offers examples for the other currencies supported natively in OSE SRD, Platinum (PP), Electrum (EP), Silver (SP), Copper (CP). This is mentioned in the OSE SRD once. Spanish i18n contributor has different names for these than us (no other locales have OSE Compendiums).

What I would suggest then is we make some i18n strings for PP, EP, SP, and CP, and set the currency names to these. We already have 1-2 strings for GP.

Reply:
I'm happy to do that myself, or @Haxxer if you'd just like to make the one change of `"Gold Pieces"` to use the i18n string `"OSE.items.gp.long"`, we'd add the others later

Reply:
Are item name localizations processed? I don't believe I do it anywhere in Item Piles.

Reply:
Thanks, will review in a few hours! If I understand correctly we now use the localized name in character generation, so when the GP item is created it uses the localized name. I just remembered we also have a "shopping cart" feature that checks for the localized name in the character's inventory and deducts the appropriate quantity upon "checkout"

Reply:
Maybe this is another case where we can use hooks to give modules the ability to modify settings? We could add additional currencies on the SRD module side, and leave this as is.

Reply:
I have made the gold pieces item localized now :)